doh.c

```diff
@@ -594,7 +632,7 @@ static DOHcode doh_decode(unsigned char *doh,
   if (index != dohlen)
     return DOH_DNS_MALFORMAT; /* something is wrong */
 
-  if ((type != DNS_TYPE_NS) && !d->numcname && !d->numv6 && !d->numv4)
+  if ((type != DNS_TYPE_NS) && !d->numcname && !d->numv6 && !d->numv4 && !d->numtxt)
```

Comment:
This `if()` should probably not have a space before the parenthesis for consistency.

Reply:
https://github.com/curl/doh/pull/27/commits/18b00d5f2e12059cad489e9f998cc39f937afdea resolved here. I did a quick scan of the whole file.